### PR TITLE
John conroy/update home page title

### LINKED ISSUE #1315: Change title of portal homepage
The title (HTML <title> element) of the portal homepage should be "HuBMAP Data Portal". Currently, the title is "Welcome | HuBMAP", which doesn't look great in Google search results.

image

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #1315.

### CODE REVIEW

File: context/app/routes_main.py [hunk 1]
```diff
@@ -42,7 +42,8 @@ def index():
         'pages/base_react.html',
         types=types,
         flask_data=core_props,
-        title='Welcome'
+        title='HuBMAP Data Portal',
+        isHomePage=True
```
Comment (mccalluc) [inline context/app/routes_main.py:46]:
```suggestion
        is_home_page=True
```

File: context/app/templates/pages/base_react.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   
     {% block extra_stylesheets %}{% endblock %}
 
-    <title>{% block title_text %}{{ title }}{% endblock %} | HuBMAP</title>
+    <title>{% block title_text %}{{ title }}{% endblock %}{{' | HuBMAP' if not isHomePage}}</title>
```
Comment (mccalluc) [inline context/app/templates/pages/base_react.html:14]:
```suggestion
    <title>{% block title_text %}{{ title }}{% endblock %}{{' | HuBMAP' if not is_home_page}}</title>
```

**Review by mccalluc — COMMENTED** [review]
Looks good... I think we decided that we would be prefer python underscore naming conventions leaking into JS, rather than the reverse? But do what you think best.

File: context/app/routes_main.py (hunk 1 above)
Comment (john-conroy) [reply context/app/routes_main.py:46]: The casing was just reflex and not intentional.

**Pull request merged**